plugin/trino-hudi/src/test/resources/hudi-testing-data/hudi_comprehensive_types_mor/README.md

```diff
@@ -0,0 +1,168 @@
+## Create script
+
+Structure of table:
+- MOR table with MDT enabled
+- Revision: 444cac26cb1077fd2b7deefc7b3713bacb270f9c
+
+```scala
+test("Create MOR table with comprehensive types") {
+    withTempDir { tmp =>
+        val tableName = "hudi_type_test_mor"
```

Comment:
Should we add tests on both table version 6 and 8 for reader compatibility?

Reply:
If there are any issues raised, don't have to fix it, just surface them.